tm2/pkg/bft/rpc/lib/types/types.go

```diff
@@ -194,7 +194,7 @@ func NewRPCSuccessResponse(id JSONRPCID, res any) RPCResponse {
 
 	if res != nil {
 		var js []byte
-		js, err := amino.MarshalJSON(res)
+		js, err := json.Marshal(res)
```

Comment:
Can we figure out why the Amino JSON is causing an empty object? What is the type of res?
Maybe we just need to register the package / type with Amino

It's now causing tests to fail, most likely because we use Amino JSON to unmarshal all responses / requests